### DISCUSSION TITLE
Need help with Power Query and Stats in Excel

### DISCUSSION THREAD

**Discussion by tehniemer** [advplyr/audiobookshelf#3409] [Q&A] [answered]
I've followed the instructions from https://www.audiobookshelf.org/guides/power-query but I'm not getting any results.

This is what my workbook looks like after I complete all the steps
![image](https://github.com/user-attachments/assets/407142f8-2550-4127-95df-5ad719747de5)

And what the Power Query Editor looks like
![image](https://github.com/user-attachments/assets/f4e42114-a86e-492f-9ef4-c01a389d0421)

Is there a missing step or did I do something wrong?

**Comment by advplyr** [comment]
The guides are user contributed so I haven't actually set that up before. The author of the guide may be able to assist. https://github.com/audiobookshelf/audiobookshelf-web/pull/98 @SAS-1

**Comment by SAS-1** [comment, marked as answer]
yeah looks like Microsoft have tweaked it - it should open up the Advanced Editor when you select Blank Query but it's not.

After you select Blank Query a window opens

![image](https://github.com/user-attachments/assets/e89ff2cf-3f77-46fb-8f9e-694b13954576)

click on the advanced editor and in the box that appears delete the text already there and paste the copied code in 

@advplyr  I did a quick PR to fix this thanks!

**Comment by tehniemer** [comment]
That did it, thank you!
